grpc-proto/src/main/java/io/stargate/grpc/Values.java

```diff
@@ -50,6 +56,18 @@ public static Value of(float value) {
     return Value.newBuilder().setFloat(value).build();
   }
 
+  public static Value of(short value) {
+    return Value.newBuilder().setInt(value).build();
+  }
+
+  public static Value of(byte value) {
+    return Value.newBuilder().setInt(value).build();
+  }
+
+  public static Value of(BigInteger value) {
```

Comment:
Good point, however, I don't see it being supported in the ` io.stargate.grpc.codec.cql.ValueCodecs`
Should I add support for it in this PR? Or is the `BigInteger` support not important for now?

Reply:
Not important for now. `varint` and `decimal` were planned for later.

Reply:
Yes, because it can silently lose information and the overload is better suited to be stored in a `varint`.  If a user wants to store the lower bits a `BigInteger` in a `bigint` they should use `Values.of(value.longValue())` explicitly.

Reply:
done